lib/content-services/src/lib/dialogs/dialog/dialog.component.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Inject, ViewEncapsulation } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { DialogComponentData } from './dialog-data.interface';
+import { BehaviorSubject } from 'rxjs';
+import { DialogSize } from './dialog.model';
+
+@Component({
+    selector: 'adf-dialog',
+    templateUrl: './dialog.component.html',
+    styleUrls: ['./dialog.component.scss'],
+    encapsulation: ViewEncapsulation.None
```

Comment:
If we're adding a new component I think it should be standalone as we're migrating most of them either way

Reply:
done, thanks